pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-events/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pidgin, glib, autoreconfHook, pkgconfig, intltool }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-events";
+  version = "2017-08-25";
```

Comment:
Why not use https://github.com/sardemff7/purple-events/releases/tag/v0.99.1 ? It's only 4 commits behind `master`: https://github.com/sardemff7/purple-events/compare/v0.99.1...master